.mk/kind.mk

```diff
@@ -35,7 +39,9 @@ kind-start: kind-config
 
 .PHONY: kind-export-kubeconfig
 kind-export-kubeconfig:
-	@kind get kubeconfig --name $(KIND_CLUSTER_NAME) > $(KIND_CLUSTER_NAME)-kubeconfig
+	@touch $(CONFIG_LOCATION); \
+	kind get kubeconfig --name $(KIND_CLUSTER_NAME) > $(CONFIG_LOCATION); \
+	echo "$(CONFIG_LOCATION)"
```

Comment:
I don't think the echo will actually show the right CONFIG_LOCATION value because it's going to re-exec the `tempdir` command which will generate a new dir.

Reply:
Good point! I will delete that line.

Reply:
Why can't I reuse the `KIND_CLUSTER_NAME` var? 

I found those changes to the `kind.mk` file in the upstream networkservicemesh/cloudtest repo: https://github.com/networkservicemesh/cloudtest/blob/master/examples/kind/Makefile#L39

Reply:
I'm thinking that `$(rands10)` generates a random string each time it's invoked.  But it's not a standard executable so maybe cloudtest does some kind of replace on it.  Craziness... but I guess if it's the established pattern in cloudtest I'm ok with it for now until we figure out whether it's a problem.